runelite-client/src/main/java/net/runelite/client/plugins/chatfilter/ChatFilterConfig.java

```diff
@@ -63,4 +64,37 @@ default String filteredRegex()
 	{
 		return "";
 	}
+
+	@ConfigItem(
+		keyName = "filterYourself",
+		name = "Filter Yourself",
+		description = "Filter your own messages",
+		position = 4
+	)
+	default boolean filterYourself()
```

Comment:
I think this was just overlooked to begin with and probably shouldn't be an option at all, and should just never filter self.

Reply:
done